### ISSUE TITLE
Possible Bug

### ISSUE THREAD

**Issue by NullMagic** [simeonradivoev/MatterOverdrive#137]
Alright, I'm not entirely 100% sure that this bug is yours, so sorry in advance if this is a wild goose chase.

So here's the Error Report: http://pastebin.com/hLR0J5eN

I'm sure you can see from the above that it was... shall we say less than explicit about what is to blame for the exception that was thrown. Which has had me running around for the last two days pulling mods until it finally stopped happening, which lead me to Matter Overdrive.

This particular crash occurred with a frequency between ten and thirty minutes after a player would log in, subsequent attempts to log in would either yield the error again or repeat the pattern (a short time of gameplay and then the crash again), more frustrating still, it's a client side crash, taking them out to the launcher but leaving the server running, and it does not seem to effect all players equally (some players wouldn't crash more than once or twice, others nearly constantly).

After removing the Matter Overdrive mod the crashes stopped, and things ran smoothly for a test period of two hours, after which the mod was added back in and allowed to gen a new config file (just in case of a corrupted one). Within 20 minutes another crash occurred.

Now, this is obviously not conclusive proof, but it is I feel suggestive, and unfortunately the biggest (and in fact only) lead we have, so I thought I'd post it here, and let you see if it rings any bells, I'm going to go turn Android spawning off and see what happens next, I'll let you know if that fixes anything.

**Comment by simeonradivoev** [comment]
what version of forge are you using and does the crashing occur when you are an Android?

**Comment by shadowfacts** [comment]
o_O
That's uh... interesting...

**Comment by NullMagic** [comment]
we are currently using forge 10.13.4.1492 universal. After turning off android spawning we were able to play for several hours without incident, I transformed myself into an android and spent a good portion of that time as one without incident (although for some reason it wouldn't let me take the damage upgrade, but I was in creative at the time so it may have had something to do with that). I'll let you know if anything else comes up, and thank you for looking into this.

**Comment by simeonradivoev** [comment]
1e9a2596f7b98e13790453066c50870e635a2165 may be a possible solution.

**Comment by NullMagic** [comment]
So, I have good news and I have bad news.

The Good news is that bug came back and we were able to track it to it's source (a completely different mod, that somehow managed to continue crashing us even after we took it out last time, which doesn't make a ton of sense, but welcome to minecraft)

The bad news is this means it's not your bug, and the fact that we somehow magically stopped having the bug for two days is complete and utter coincidence, I'm terribly sorry about that, since you got on this immediately and started looking for the issue and how to fix it. I feel pretty bad that I wasted your time, so, sorry about that. And thank you for your quick action and hard work.
